src/include/gcc.h

```diff
@@ -203,15 +203,15 @@ WT_ATOMIC_FUNC(size, size_t, size_t *vp, size_t v)
 #define WT_PAUSE() __asm__ volatile("yield" ::: "memory")
 #define WT_FULL_BARRIER()                        \
     do {                                         \
-        __asm__ volatile("dsb sy" ::: "memory"); \
```

Comment:
It would be great if you can add the content in the description on the PR as a comment above here, so we can remember why the `ish` operators are enough.